diagrams/aws/analytics.py

```diff
@@ -104,6 +104,10 @@ class ManagedStreamingForKafka(_Analytics):
     _icon = "managed-streaming-for-kafka.png"
 
 
+class Opensearch(_Analytics):
+    _icon = "opensearch.png"
+
+
```

Comment:
Should be OpenSearch with the file open-search.png

when checking on my AWS console the OpenSearch top page display the name with a S in capital case. 

"Amazon OpenSearch Service"